crates/vision/src/camera_matrix_extractor.rs

```diff
@@ -13,8 +11,9 @@ pub struct CreationContext {}
 #[context]
 pub struct CycleContext {
     pub camera_matrices: RequiredInput<Option<CameraMatrices>, "Control", "camera_matrices?">,
+    pub camera_position:
+        Parameter<CameraPosition, "image_receiver.$cycler_instance.camera_position">,
     pub image: Input<YCbCr422Image, "image">, // required for correct node order
```

Comment:
Remove `image` input because you have now the concept of setup nodes?

Reply:
Done